src/puppetlabs/puppetdb/command.clj

```diff
@@ -529,12 +534,15 @@
 (def quick-retry-count 4)
 
 (defn attempt-exec-command
-  [{:keys [callback] :as cmd} db conn-status response-pub-chan stats-atom
+  ([{:keys [callback] :as cmd} db conn-status response-pub-chan stats-atom
    shutdown-for-ex]
+   attempt-exec-command cmd db conn-status response-pub-chan stats-atom shutdown-for-ex {})
+  ([{:keys [callback] :as cmd} db conn-status response-pub-chan stats-atom
+   shutdown-for-ex blocklist-config]
   (try
     (let [result (call-with-quick-retry quick-retry-count
                                         shutdown-for-ex
-                                        #(exec-command cmd db conn-status (now)))]
+                                        #(exec-command cmd db conn-status (now) blocklist-config))]
```

Comment:
We might want to rename `blocklist-config` to something more general now that it's holding blocklist settings and resource-events-ttl.

Reply:
changed to `options-config`